ext/opentelemetry-ext-redis/src/opentelemetry/ext/redis/util.py

```diff
@@ -0,0 +1,55 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+Some utils used by the redis integration
+"""
+
+
+def _extract_conn_attributes(conn_kwargs):
+    """ Transform redis conn info into dict """
+    try:
+        return {
+            "db.type": "redis",
```

Comment:
should this be added regardless?

Reply:
yes, updated the code to set available attributes